### PR TITLE
Support for missing dashbot properties

### PR DESCRIPTION
**Pull request by omenocal**
Just adding the `facebook` key to the `IVoxaDashbotConfig` interface

### CODE REVIEW

File: src/Voxa-Dashbot.ts [hunk 1]
```diff
@@ -50,6 +55,8 @@ export function register(skill: VoxaApp, config: IVoxaDashbotConfig) {
   const dashbotConfig = {
     debug: pluginConfig.debug,
     printErrors: true,
```
Comment (armonge) [inline src/Voxa-Dashbot.ts]: should this be a configuration option?

File: src/Voxa-Dashbot.ts [hunk 2]
```diff
@@ -41,7 +43,10 @@ export interface IVoxaDashbotConfig {
   botframework?: string;
   debug?: boolean;
   dialogflow?: string;
+  facebook?: string;
+  redact?: boolean;
   suppressSending?: boolean;
```
Comment (armonge) [inline src/Voxa-Dashbot.ts]: one thing i noticed with our latest project, is that we need a way to make this work as a function, something that checks the userId and decides to send or not based on that.

With loader.io for example we used big range or user ids, we want to be able to not send analytics in that case, for that range of usrs. Maybe for other PR?

Comment (omenocal) [reply src/Voxa-Dashbot.ts]: @armonge I think that's why we have this property: `ignoreUsers`, it's an array of userIds that should be ignored and analytics should not be send if the userId is among those ones

Comment (armonge) [reply src/Voxa-Dashbot.ts]: Yes, but in the case i'm talking about, the list was too large to be included in a configuration array, 

```
        "userId": "ABwppHEKYraiAED6VdrL3RrtqFcjyC7R1NOCaqDh9WUnSMjlMYNyG9gHyN9iDWEjV1Gl_S_K3_%{*:10000-99999}"
```

Comment (omenocal) [reply src/Voxa-Dashbot.ts]: Understood, I can look into that for another PR 👍

**Pull request merged**